Index: src/BulletSoftBody/btSoftBodyInternals.h
===================================================================
--- src/BulletSoftBody/btSoftBodyInternals.h	(revision 2286)
+++ src/BulletSoftBody/btSoftBodyInternals.h	(working copy)
@@ -25,7 +25,7 @@
 #include "BulletCollision/CollisionDispatch/btCollisionDispatcher.h"
 #include "BulletCollision/CollisionShapes/btConvexInternalShape.h"
 #include "BulletCollision/NarrowPhaseCollision/btGjkEpa2.h"
-
+#include <string.h> //for memset
 //
 // btSymMatrix
 //
@@ -172,8 +172,7 @@
 template <typename T>
 static inline void			ZeroInitialize(T& value)
 {
-	static const T	zerodummy;
-	value=zerodummy;
+	memset(&value,0,sizeof(T));
 }
 //
 template <typename T>
